.eslintrc

```diff
@@ -3,6 +3,7 @@
     "plugins": ["jest"],
     "env": {
         "es6": true,
+        "jasmine": true,
```

Comment:
Note for reviewers: this is for the linter to recognize the `fail()` syntax and not complain that it's undefined.